### ISSUE TITLE
Hunter pet's pathing 

### ISSUE THREAD

**Issue by chiolol** [SunwellTracker/issues#3004]
Dec:
I want to let you know that the pathing of pets is insane. I am playing hunter for many years and I have never seen anything like that before. Pets are totally useless. 
Unfortunately, high elo BM is unplayable atm.  Just have a look. Thank you for your time!
See how the pet moves.

How it works: 
The pet move back and forth

How it should work: 
The pet must move smoothly forward.

Source (you should point out proofs of your report, please give us some source): 

https://www.youtube.com/watch?v=35vSzlX-5fA&feature=youtu.be

**Comment by jcd560** [comment]
I have created another thread as the developers seem to lack the ability to address and fix fundamental problems with the core of the hunter-experience post level 10.

This is an over-all fun-ruining, shitshow of an experience from level 10 onwards. Not to mention my hunter is shooting while moving 75% of fights, shooting targets that are hitting me with melee attacks, etc. 

This is a fundamental issue that NEEDS to be fixed in order for hunters to feel AT-ALL playable. How they can charge people for items, boosts, and other stuff while fucking up such an over-arching piece of gameplay for an entire class is so beyond my ability to perceive.

**Comment by jcd560** [comment]
Update : According to GM, this should no longer be an issue.

Bitching was ultimately a successful endeavor. 

TL;DR : Pet Pathing was bad, it's supposedly not anymore. GG EZ
